hub/src/server/authentication.js

```diff
@@ -81,9 +83,15 @@ export class V1Authentication {
       validateScopes(scopes)
     }
 
+    let payloadCreationDate = (Date.now()/1000|0)
+    if (creationDate){
+      payloadCreationDate = creationDate
+    }
+
     const payload: TokenPayloadType = { gaiaChallenge: challengeText,
                       iss: publicKeyHex,
                       exp: FOUR_MONTH_SECONDS + (new Date()/1000),
+                      creationDate: payloadCreationDate,
```

Comment:
Can we use standard JWT fields like `iat` for "issued at" and `nbf` for "not used before" to encode this? 
 Full list here:  https://en.wikipedia.org/wiki/JSON_Web_Token